Fix: search history text overflows if search term is long

ozrendev opened the pull request:
# Fix search history text overflows if search term is long

<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines. -->
<!-- Important note, we may remove your pull request if you do not use this provided PR template correctly. -->

## Pull Request Type
<!-- Please select what type of pull request this is: [x] -->
- [x] Bugfix
- [ ] Feature Implementation
- [ ] Documentation
- [ ] Other

## Related issue
closes #6727 

## Description
Truncates text that exceeds the length of the search history input

absidue commented:
When making UI changes it is always a good idea to provide before and after screenshots. Please re-add the parts of the template that you removed and add screenshots to it.

efb4f5ff-1298-471a-8973-3d47447115dc reviewed (commented):
When we wanted to introduce this feature i pointed out that the users werent able to see the full search query, [Unable to see full search suggestion](https://github.com/FreeTubeApp/FreeTube/pull/6414#issuecomment-2564401808)

[Reply](https://github.com/FreeTubeApp/FreeTube/pull/6414#issuecomment-2564420542)

By fixing the overflow you introduce the issue i pointed out before

![VirtualBoxVM_p5c7tA4xB7](https://github.com/user-attachments/assets/7986c4f3-e6a9-4009-aa1a-f242231ae667)

ozrendev commented:
> When we wanted to introduce this feature i pointed out that the users werent able to see the full search query, [Unable to see full search suggestion](https://github.com/FreeTubeApp/FreeTube/pull/6414#issuecomment-2564401808)

Okay I wasn't aware that was an issue. Was there a decision made about how it should look?

This is what is looks like when the text contained and wrapped. Otherwise I guess you can keep the ellipsis cut off but have an alt text/tooltip when hovered over?
![2025-02-02_17-02-19](https://github.com/user-attachments/assets/064e1676-0722-4a36-91ff-06a437c34f86)

ozrendev commented:
I've removed the ellipsis and adjusted it to look like before. Although I think the icon is a few pixels off, it still looks very similar.

![2025-02-12_20-29-21](https://github.com/user-attachments/assets/2543c486-d36c-4647-ae8f-80034809d873)

efb4f5ff-1298-471a-8973-3d47447115dc reviewed (approved):
LGTM

ozrendev commented on src/renderer/components/ft-input/ft-input.css:
```diff
@@ -210,6 +210,16 @@ body[dir='rtl'] .ft-input-component.search.showClearTextButton:focus-within .inp
+.optionWrapper {
+  display: flex;
+  align-items: baseline;
+  inline-size: 85%;
```
Yeah, it's not a very robust solution. The latest commit should fix it.

![2025-02-18_11-20-14](https://github.com/user-attachments/assets/cb75d916-2909-4851-bec2-c0afd797b355)
